src/gui/MainWindow.cpp

```diff
@@ -1642,7 +1642,14 @@ void MainWindow::MovableQMdiArea::mouseMoveEvent(QMouseEvent* event)
 	scrollY = scrollY < 0 && minY >= minYBoundary ? 0 : scrollY;
 	scrollY = scrollY > 0 && maxY <= maxYBoundary ? 0 : scrollY;
 
-	scrollContentsBy(-scrollX, -scrollY);
+	for (auto* curWindow : subWindows)
+	{
+		if (curWindow->isVisible() && curWindow->isMaximized() == false)
+		{
+			curWindow->move(curWindow->x() - scrollX, curWindow->y() - scrollY);
```

Comment:
A quick glance shows that `scrollX` is calculated from `minX`, which also has a loop and a conditional that are nearly identical to the code added in this PR.  I feel this logic can be simplified.

Reply:
I think I read the code too quickly.  The second loop needs the value from the first.